test/test_chronos.py

```diff
@@ -174,20 +175,20 @@ def test_pipeline_predict(torch_dtype: str):
     # input: tensor of shape (batch_size, context_length)
 
     samples = pipeline.predict(context, num_samples=12, prediction_length=3)
-    validate_tensor(samples, (4, 12, 3))
+    validate_tensor(samples, shape=(4, 12, 3), dtype=torch.float32)
```

Comment:
The output is always 32-bit float? Do we want that? Would be nicer if the output has the same type as input.

Reply:
Maybe yes. My initial reasoning was that, we do that differently when it comes to shapes: the output is always a 3D tensor, regardless of the input layout (1D tensor, list of 1D tensors, 2D tensor). But I'm fine either way.